### PR TITLE
[build-script] Fix --enable-tsan-runtime option

### PR DESCRIPTION
**Pull request by Rostepher**
Changed the --enable-tsan-runtime option to use the enable action like the surrounding tsan options.

### CODE REVIEW

**Comment by Rostepher** [comment]
@swift-ci please smoke test

File: utils/build_swift/driver_arguments.py [hunk 1]
```diff
@@ -972,10 +972,6 @@ def create_argument_parser():
         help="the absolute path to libtool. Default is auto detected.",
         type=arguments.type.executable,
         metavar="PATH")
-    parser.add_argument(
-        "--distcc",
```
Comment (gparker42) [inline utils/build_swift/driver_arguments.py]: What is this change?

Comment (Rostepher) [reply utils/build_swift/driver_arguments.py]: I was testing something out locally and mistakenly removed this option. I've added it back.

**Comment by Rostepher** [comment]
@swift-ci please smoke test

File: utils/build_swift/driver_arguments.py [hunk 2]
```diff
@@ -990,7 +989,8 @@ def create_argument_parser():
         action=arguments.action.enable)
     parser.add_argument(
         "--enable-tsan-runtime",
-        help="enable Thread Sanitizer on the swift runtime")
+        help="enable Thread Sanitizer on the swift runtime",
+        action=arguments.action.enable)
```
Comment (zisko) [inline utils/build_swift/driver_arguments.py]: please put help at the last argument here

**Review by zisko — CHANGES_REQUESTED** [review]

File: utils/build_swift/driver_arguments.py (hunk 2 above)
Comment (Rostepher) [reply utils/build_swift/driver_arguments.py]: I'm not going to do this at the moment. I have that change in a follow-up PR to #12873

**Review by zisko — APPROVED** [review]

**Pull request merged**